_summaries/zhao2020point_2.md

```diff
@@ -1,19 +1,68 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
----
+title: Point Transformer
+paper: Zhao2020point
+author: # sritank
+score: # 9/10
+
+* Core idea?
+
+They design a Point Transformer layer for point cloud processing suitable for point cloud labeling and segmentation. They introduce vector self-attention and trainable position encoding among others to make their residual transformer network invariant to permutation and cardinality. 
+
+* Technical Implementation?
+
+Given a set of feature vectors {x_i}, the vector self-attention function can be written as below:
+
+![Vector attention](zhao2020point_2a.png)
+
+where alpah(x) is the transformed features and gamma is the attention vector. They apply self attention locally with a neighbourhood of each datapoint in the cloud. The point transformer layer is created as follows:
+
+![Point transformer layer](zhao2020point_2b.png)
+
+They define their position encoding based on the point coordinates multiplied with trainable parameters as below:
+
+![Position encoding](zhao2020point_2c.png)
+
+The vector theta is an MLP which optimizes the encoding parameters. 
+
+To reduce the cardinality of the point set, they introduce transition stages. At each transition stage they do a farthest point sampling to identify the subset of the desired cardinality. To do dense predicitions they introduce
+a transition up step which maps the lower set to the higher order set using trilinear interpolation. The transition functions are illustrated below:
+
+![Transition functions](zhao2020point_2d.png)
+
+Finally they use a residual point transformer block which has a point transformer layer as shown below:
+
+![REsidual point transformer](zhao2020point_2e.png)
+
+Overall the network structure is as below:
+
+![Network](zhao2020point_2f.png)
+
+
+* Model Performance
+
+They test their model on 3D semantic segmentation and shape classification tasks. 
+
+For Segmentation tasks, the Point Transformer outperformed all prior models such as PointNet in all metrics. It even performed better than voxel based architectures such as SegCloud.
+
+![Seg expts](zhao2020point_2g.png)
+
+For shape classification, the PointTransformer outperformed all prior classifiers and achieved SOTA results as shown below:
+
+![classification](zhao2020point_2h.png)
+
+![examples](zhao2020point_2i.png)
+
+
+* Variants
+
+The authors tried using relative and absolute position encodings and found relative positions to work the best.
+
+They tried using scalar attention instead of vector attention and found that vector attention outperformed by more than 6% absolute.
+
```

Comment:
Tables 6 and 7 from the paper show the ablation study results for both the position encoding and self-attention operator. I recommend adding this table.